cmd/cdi-controller/controller.go

```diff
@@ -142,6 +161,10 @@ func start(cfg *rest.Config, stopCh <-chan struct{}) {
 	go pvcInformerFactory.Start(stopCh)
 	go podInformerFactory.Start(stopCh)
 	go serviceInformerFactory.Start(stopCh)
+	go ingressInformerFactory.Start(stopCh)
+	if isOpenshift:= controller.IsOpenshift(client) ; isOpenshift {
+		go routeInformerFactory.Start(stopCh)
```

Comment:
if we start this only if it's openshift, why don't we set routeInformerFactory,  routeInformer also conditionally?